spec/index.bs

```diff
@@ -2107,8 +2114,14 @@ A {{GPUBindGroup}} object has the following internal slots:
             This |layoutBinding| must be compatible with this |bufferBinding|. This requires:
             1. If |layoutBinding|.{{GPUBindGroupLayoutEntry/type}} is {{GPUBindingType/"uniform-buffer"}},
                 the |bufferBinding|.{{GPUBufferBinding/buffer}}'s {{GPUBufferDescriptor/usage}} must include
-                {{GPUBufferUsage/UNIFORM}}. The buffer is added to the {{GPUBindGroup/[[usedBuffers]]}} map
-                with {{GPUBufferUsage/UNIFORM}} flag.
+                {{GPUBufferUsage/UNIFORM}} and |bufferBinding|.{{GPUBufferBinding/size}} must be less than or equal
+                {{GPULimits/maxUniformBufferBindingSize}}. The buffer is added to the {{GPUBindGroup/[[usedBuffers]]}}
+                map with {{GPUBufferUsage/UNIFORM}} flag.
+
+                Issue: This validation should take into account the default when  {{GPUBufferBinding/size}} is not set.
+                Also should {{GPUBufferBinding/size}} default to the `buffer.byteLength - offset` or
```

Comment:
If you default to `min(buffer.byteLength - offset, limits.maxUniformBufferBindingSize)` would it always pass validation when it shouldn't?

Reply:
We just merged this but @Kangz can you check this?

Reply:
It would always pass validation but could result in clipping the data in surprising ways for the developer.